packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -285,33 +296,36 @@ const PaymentMethodTitle: FunctionComponent<
     };
 
     return (
-        <div className="paymentProviderHeader-container">
-            <div
-                className="paymentProviderHeader-nameContainer"
-                data-test={`payment-method-${method.id}`}
-            >
-                {logoUrl && (
-                    <img
-                        alt={methodName}
-                        className="paymentProviderHeader-img"
-                        data-test="payment-method-logo"
-                        src={logoUrl}
-                    />
-                )}
+        <>
+            <div className="paymentProviderHeader-container">
+                <div
+                    className="paymentProviderHeader-nameContainer"
+                    data-test={`payment-method-${method.id}`}
+                >
+                    {logoUrl && (
+                        <img
+                            alt={methodName}
+                            className="paymentProviderHeader-img"
+                            data-test="payment-method-logo"
+                            src={logoUrl}
+                        />
+                    )}
 
-                {titleText && (
-                    <div aria-level={6} className="paymentProviderHeader-name" data-test="payment-method-name" role="heading">
-                        {titleText}
-                    </div>
-                )}
-            </div>
-            <div className="paymentProviderHeader-cc">
-                <CreditCardIconList
-                    cardTypes={compact(method.supportedCards.map(mapFromPaymentMethodCardType))}
-                    selectedCardType={getSelectedCardType()}
-                />
+                    {titleText && (
+                        <div aria-level={6} className="paymentProviderHeader-name" data-test="payment-method-name" role="heading">
+                            {titleText}
+                        </div>
+                    )}
+                </div>
+                <div className="paymentProviderHeader-cc">
+                    <CreditCardIconList
+                        cardTypes={compact(method.supportedCards.map(mapFromPaymentMethodCardType))}
+                        selectedCardType={getSelectedCardType()}
+                    />
+                </div>
             </div>
-        </div>
+            {subtitle instanceof Function ? subtitle({ onUnhandledError }) : subtitle}
```

Comment:
I'm concerned that putting `subtitle` outside of `paymentProviderHeader-container` would cause merchants' existing CSS customizations to break. It also breaks the logic of the checkout's dom-structure.
Is it possbile to keep using `paymentProviderHeader-container` to hold everything?

Reply:
Updated with using `flex-basis`